virttest/block.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility functions to handle block related actions.
+
+"""
+
+from . import qemu_monitor
+
+
+def get_block_virtual_size(dev_dict):
+    """
+    Get block virtual-size of specified block device
+    from monitor's info block command.
+
+    :param dev_dict: define device dict from block info.
+    :return: Matched block virual size.
+    """
+    blocks_info = qemu_monitor.info("block")
+    for dict_block in blocks_info:
+        if dev_dict['device'] in dict_block['device']:
+            size = dict_block['inserted']['image']['virtual-size']
+    if size:
+        return size
```

Comment:
It's better to handle size=none situation

Reply:
Hi xiagao, has updated code, could you help to review it again?